web/policies/donation_goal_policy.ex

```diff
@@ -0,0 +1,40 @@
+defmodule CodeCorps.DonationGoalPolicy do
```

Comment:
Admin user can create, update, delete
Admin or higher in project's organization can create, update, delete

Authenticated user can view